spec/features/idv/previous_address_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'IdV with previous address filled in' do
+  include IdvHelper
+
+  let(:bad_zipcode) { '00000' }
+
+  it 'fails when either address has bad value, prefers current address in profile' do
+    user = sign_in_and_2fa_user
+    visit verify_session_path
+    fill_out_idv_form_ok
+    fill_out_idv_previous_address_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
+
+    fill_out_idv_previous_address_ok
+    fill_out_idv_form_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
+
+    fill_out_idv_form_ok
+    fill_out_idv_previous_address_ok
+    click_idv_continue
+
+    expect(current_path).to eq verify_finance_path
+
+    fill_out_financial_form_ok
+    click_idv_continue
+
+    fill_out_phone_form_ok(user.phone)
+    click_idv_continue
+
+    expect(current_path).to eq verify_review_path
+    expect(page).to have_content('123 Main St')
```

Comment:
can we label this `current_address` or something so it's obvious why this expectation is correct? had to scroll up to the spec description a few times to understand why we would want to see one address or the other.